ui/admin/tests/e2e/README.md

```diff
@@ -1,36 +1,133 @@
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 # Admin UI e2e tests
 
 This test suite tests the Boundary Admin UI in an end-to-end setting. It was designed to be run in a
 variety of environments as long as the appropriate environment variables are set. You can choose to
 spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
-to generate it for you.
+to generate it for you. More information about [Enos available here.](https://github.com/hashicorp/Enos-Docs)
 
 The test suite uses the [Playwright](https://playwright.dev/) framework.
 
+## Prerequisites:
+
+You will need [Hombrebrew](https://brew.sh/) install. For secure persisting your SSH keys and tokens we recommend using 1Password.
+
+### Accesses
+
+If you are missing any acccess, requested [through the IT service catalog](https://hashicorp.freshservice.com/support/catalog/items).
+
+- Doormat account: `boundary_team_acctest_dev`.
+- HCP account: you will need to use Terraform cloud to spin up enos.
+
+### Software
+
+- Vault: [documentation how to install](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-install).
+- Terraform: [documentation how to install](https://developer.hashicorp.com/terraform/downloads).
+- Github CLI: [documentation how to install](https://cli.github.com/manual/installation).
+- Install Enos: [documentation how to install using homebrew](https://github.com/hashicorp/Enos-Docs/blob/main/installation.md), also binaries available [here](https://github.com/hashicorp/enos/releases).
+- Install Doormat CLI: [documentation how to install](https://docs.prod.secops.hashicorp.services/doormat/cli/).
+- Install AWS CLI (Optional): [documentation how to install](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
 ## Getting Started
-Setup...
-```bash
-cd boundary-ui/ui/admin
-yarn install
-npx playwright install # this installs the browsers used by Playwright
-```
+
+we will cover everything you need to setup before be able to run Enos + the tests.
+
+### Setup Boundary CLI
 
 Ensure the `boundary` CLI is available on the path. You can validate this by making sure this
 command returns something.
 ```bash
 which boundary
 ```
 
-Run tests...
+Otherwise, follow instructions to [install Boundary](https://developer.hashicorp.com/boundary/downloads?product_intent=boundary) or install it locally running `$ make install` within [Boundary repository](https://github.com/hashicorp/boundary/tree/main).
+
+### Setup Boundary UI
+
+Ensure you install project dependencies and playwright needs.
+
 ```bash
-export BOUNDARY_ADDR=
-export E2E_PASSWORD_ADMIN_LOGIN_NAME=
-export E2E_PASSWORD_ADMIN_PASSWORD=
-export E2E_PASSWORD_AUTH_METHOD_ID=
-... # Tests may have additional variables that it needs
 cd boundary-ui/ui/admin
-yarn run e2e
+yarn install
+npx playwright install # this installs the browsers used by Playwright
 ```
+### Setup EC2 instance:
```

Comment:
To be clear, I don't think we're actually setting up an ec2 instance itself here